src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1144,19 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from parameter string
```

Comment:
good to see you have updated the header comments as well :-)
however, the use of "parameter string" still makes it specific to address book. please update that as well to make it generic.